### PR TITLE
Updates based on discussion

### PR DESCRIPTION
**Pull request by scharch**
Most of the changes are based on the consensus of discussions with @eharkins and @javh. A few other minor things:
* I added `repertoire_id` and `data_processing_id` as required in `Clone`
* Do we need explicit `required: false` tags the way we seem to be doing for `miairr`?
* Is list a valid `type` for `sequence_id` in `Clone`?

### CODE REVIEW

**Comment by scharch** [comment]
@eharkins @javh I don't have write permissions.... TIA

**Comment by eharkins** [comment]
Thanks @scharch! Not sure but I assume I don't have the authority to give you permissions to the airr-standards repo if that's what you mean?

> I added repertoire_id and data_processing_id as required in Clone

👍 
> Do we need explicit required: false tags the way we seem to be doing for miairr?

I didn't add any just to be consistent with existing `x-airr` fields in this schema. Up to the AIRR group but seems like it just depends on whether the schema is validating that the `required` field exists or not. 
> Is list a valid type for sequence_id in Clone

I think it should be [array](https://cswr.github.io/JsonSchema/spec/arrays/) for json schema.

**Comment by scharch** [comment]
> Not sure but I assume I don't have the authority to give you permissions to the airr-standards repo

@eharkins no, I just meant I can't commit directly to your PR, so this is a PR on your PR that you have to merge in

**Comment by eharkins** [comment]
@scharch got it. Let's see what @javh  and others have to say and then I'll gladly merge. Thanks for putting it together.

File: specs/airr-schema.yaml [hunk 1]
```diff
@@ -2267,9 +2266,16 @@ Clone:
...
                 miairr: false
+                required: true
         data_processing_id:
             type: string
             description: Identifier for the data processing object.
+            x-airr:
+                miairr: false
+                required: true
+        sequence_id:
+            type: list
```
Comment (eharkins) [inline specs/airr-schema.yaml]: @scharch can you change this to be [array as in this link](https://cswr.github.io/JsonSchema/spec/arrays/) and then I can merge?

Comment (scharch) [reply specs/airr-schema.yaml]: @eharkins done :)

Comment (eharkins) [reply specs/airr-schema.yaml]: @scharch thanks! I'll merge now. Should we add the type of items in the array like they do in the docs I linked:
```
 "items": {
           "type": "string"
          }
```
(or other restrictions on what the array can contain)?

Comment (scharch) [reply specs/airr-schema.yaml]: > Should we add the type of items in the array like they do in the docs I linked:

dunno, sounds like a question for @schristley on what our typical format/standard for such things is...

Comment (eharkins) [reply specs/airr-schema.yaml]: The jsonschema array docs suggest you might be required to specify the `items` definition, and regardless it seems like a good idea if we know the expected type of things in the array as we probably do with `sequence_id` here. Also most of the array fields in the existing AIRR schemas have an `items` definition [e.g.](https://github.com/airr-community/airr-standards/blob/master/specs/airr-schema.yaml#L1290)

I can add one directly to the merged `clones-and-trees-release` branch.

**Pull request merged**